src/composables/pools/usePoolCreation.ts

```diff
@@ -153,18 +153,21 @@ export default function usePoolCreation() {
   });
 
   const currentLiquidity = computed(() => {
-    let total = 0;
+    let total = bnum(0);
     for (const token of poolCreationState.seedTokens) {
-      total = total + priceFor(token.tokenAddress) * token.amount;
+      total = total.plus(
+        bnum(token.amount).times(priceFor(token.tokenAddress))
+      );
```

Comment:
If `token.amount` is a string, you should be able to do this:
```suggestion
      total = total.plus(token.amount).times(priceFor(token.tokenAddress));
```